templates/vue/src/main.js

```diff
@@ -5,7 +5,9 @@ import store from "./store"
 import router from "./router"
 import "bootstrap/dist/css/bootstrap.css"
 import "bootstrap-vue/dist/bootstrap-vue.css"
+import VueYouTubeEmbed from 'vue-youtube-embed'
```

Comment:
Do we have to import this here? Can we not do the import in a more specific component instead?

Reply:
The [documentation](https://github.com/kaorun343/vue-youtube-embed) doesn't present another way to do this I'm afraid.